pkg/README.md

```diff
@@ -45,5 +45,19 @@ ts-node example/test.ts
 
 ### NPM
 
-link to package on npm: https://www.npmjs.com/package/x25519
+links to packages on npm: 
```

Comment:
Everything in `./pkg` is generated code so this only shows up here because the package the readme was changed in another PR.  I actually think we should remove `./pkg` from git, but thats another issue